src/main.rs

```diff
@@ -146,8 +146,8 @@ impl Actor for Node {
 
 #[derive(Clone)]
 struct ModelCfg {
-    elder_count: usize,
-    server_count: usize,
+    initial_elders: usize,
+    final_elders: usize,
```

Comment:
not elders, it's the number of nodes in the network

Reply:
Is that for both variables? Or just `server_count`?